kloppy/infra/serializers/event/statsbomb/serializer.py

```diff
@@ -40,6 +45,13 @@
 
 SB_EVENT_TYPE_HALF_START = 18
 SB_EVENT_TYPE_HALF_END = 34
+SB_EVENT_TYPE_STARTING_XI = 35
+
+SB_EVENT_TYPE_SUBSTITUTION = 19
```

Comment:
Nevermind, I see statsbomb have both substition and on/off.  Will familiarize myself with the dataset formats.